src/geckomat/gather_kcats/readDLKcatOutput.m

```diff
@@ -31,7 +31,7 @@
 params = modelAdapter.params;
 
 if nargin<2 || isempty(outFile)
-    fID      = fopen(fullfile(params.path,'data','DLKcatOutput.tsv'),'r');
+    fID      = fopen(fullfile(params.path,'data','DLKcat.tsv'),'r');
 else
     fID      = fopen(outFile);
 end
```

Comment:
I just have an example file, but I notice that line 38 (textscan) is taking the first line as a header. In the example file (from teams) there is no headers.

Reply:
Yes, I'm about to write this, but with this PR you should redownload and reinstall DLkcat (so run `system('pipenv rm')` from the dlkcat folder and remove the folder afterwards). There are minor changes in the DLKcat I/O files, including this step here.